diff --git a/ios/Galeria.podspec b/ios/Galeria.podspec
index 1eb3e35be2209cdacbda1fa8e0bd27877373cdcc..2f64608a7492b7030351c9ccdd3e57e81ec6a0d7 100644
--- a/ios/Galeria.podspec
+++ b/ios/Galeria.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '16.0'
+  s.platform       = :ios, '16.4'
   s.swift_version  = '5.4'
   s.source         = { git: 'https://github.com/nandorojo/galeria' }
   s.static_framework = true
